sdk/search/search-documents/src/searchIndexingBufferedSenderImpl.ts

```diff
@@ -354,7 +381,12 @@ class SearchIndexingBufferedSenderImpl<T> implements SearchIndexingBufferedSende
       // raise success event
       this.emitter.emit("batchSucceeded", result);
     } catch (e) {
-      if (this.isRetryAbleError(e) && retryAttempt < DEFAULT_RETRY_COUNT) {
+      if (this.isRetryAbleError(e) && retryAttempt <= this.maxRetries) {
+        const timeToWait =
```

Comment:
I think it's better/clearer to calculate the delay and then do a Math.min() with the maximum e.g.

```ts
const timeToWait = Math.min(retryAttempt * this.retryDelay, this.maxRetryDelay);
```

Reply:
Done